em_workflows/utils/utils.py

```diff
@@ -522,11 +522,20 @@ def notify_api_completion(flow: Flow, flow_run: FlowRun, state: State):
     https://docs.prefect.io/core/concepts/states.html#overview.
     https://docs.prefect.io/core/concepts/notifications.html#state-handlers
     """
-    status = "Completed" if state.is_completed else "Failed"
-    x_no_api = flow_run.parameters.get("x_no_api", True)
+    status = "success" if state.is_completed() else "error"
+    x_no_api = flow_run.parameters.get("x_no_api", False)
     token = flow_run.parameters.get("token", "")
     callback_url = flow_run.parameters.get("callback_url", "")
 
+    import sys
+
+    old = sys.stdout
```

Comment:
explanation why:
currently hooks are not logged properly in slurm or prefect logs - workaround hack - will rm when no longer needed.

Reply:
Done